apps/voting-disputable/arapp.json

```diff
@@ -72,7 +72,7 @@
     },
     {
       "name": "Modify quiet ending configuration",
-      "id": "MODIFY_QUIET_ENDING_CONFIGURATION",
+      "id": "MODIFY_QUIET_ENDING_ROLE",
```

Comment:
I don't think this will have any breaking changes for us anywhere yet, but good note that we always suffix roles with `_ROLE` :)